src/fragments/lib/graphqlapi/ios/getting-started/50_createtodo.mdx

```diff
@@ -1,58 +1,17 @@
 
-<BlockSwitcher>
-
-<Block name="Listener (iOS 11+)">
-
-```swift
-func createTodo() {
-    let todo = Todo(name: "my first todo", description: "todo description")
-    Amplify.API.mutate(request: .create(todo)) { event in
-        switch event {
-        case .success(let result):
-            switch result {
-            case .success(let todo):
-                print("Successfully created the todo: \(todo)")
-            case .failure(let graphQLError):
-                print("Failed to create graphql \(graphQLError)")
-            }
-        case .failure(let apiError):
-            print("Failed to create a todo", apiError)
-        }
-    }
-}
-```
-
-</Block>
-
-<Block name="Combine">
```

Comment:
I don't think we're removing the Combine versions of the API, see Auth for example https://github.com/aws-amplify/docs/pull/4579 there's "Async/Await" that replaces "Listener (iOS 11+)" and then Combine is still there. Every APi can be wrapped with `Amplify.Publisher.create` to create the Combine publisher